File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/ActionsItem.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class ActionsItem{

	@SerializedName("openPopupAction")
	private OpenPopupAction openPopupAction;

	@SerializedName("clickTrackingParams")
	private String clickTrackingParams;

	public void setOpenPopupAction(OpenPopupAction openPopupAction){
		this.openPopupAction = openPopupAction;
	}

	public OpenPopupAction getOpenPopupAction(){
		return openPopupAction;
	}

	public void setClickTrackingParams(String clickTrackingParams){
		this.clickTrackingParams = clickTrackingParams;
	}

	public String getClickTrackingParams(){
		return clickTrackingParams;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/ActionsItem.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class ActionsItem : Parcelable, Serializable {
    @SerializedName("openPopupAction")
    var openPopupAction: OpenPopupAction? = null

    @SerializedName("clickTrackingParams")
    var clickTrackingParams: String? = null
    override fun toString(): String {
        return "ActionsItem{" +
                "openPopupAction = '" + openPopupAction + '\'' +
                ",clickTrackingParams = '" + clickTrackingParams + '\'' +
                "}"
    }
}